### ISSUE TITLE
`ImageSource` is logging forever

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno.toolkit.ui#746]
### Current behavior

![image](https://github.com/unoplatform/uno/assets/31348972/55147aee-0d3c-4a1a-b522-73db2802e90c)

This keeps being logged forever.

Repro project: https://github.com/Youssef1313/issues10285

### Works on UWP/WinUI

None

**Comment by jeromelaban** [comment]
Does https://github.com/unoplatform/uno/pull/13459 fix the issue?

**Comment by Youssef1313** [comment]
Yes, I think. Though we need the changes around unloading the ProgressRing when it's not in the visual tree. Not sure what is the status of it, but the logging itself should be better now with being debug level.
